redirects/7.1/concepts/JMX-props-config.html

```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/security/
```

Comment:
security is not a folder, it is markdown

Reply:
Please see my earlier comments - https://github.com/Alfresco/docs-alfresco/pull/749#discussion_r931313646. This is working as designed. I've resolved all the following comments as there's nothing to change. Thanks @Amy-Hyland